nucliadb_node/requirements.txt

```diff
@@ -6,6 +6,6 @@ grpcio-status>=1.44.0
 grpcio-tools>=1.44.0
 grpcio-testing>=1.44.0
 grpcio-reflection>=1.44.0
-nucliadb-telemetry>=1.15.0
+nucliadb-telemetry[all]
 nucliadb-protos>=2.24.0
```

Comment:
should these go in requirements.txt or in requirements-sources, bot not both?

Reply:
sources are loaded separately for dev than requirements.txt. requirements.txt are only used in defining the actual requirements of the package.